src/main/java/com/iota/iri/conf/DbConfig.java

```diff
@@ -54,6 +54,34 @@
      */
     boolean isRescanDb();
 
+    /**
+     * Default Value: {@value BaseIotaConfig.Defaults#TX_BATCH_WRITE}
+     *
+     * @return {@value DbConfig.Descriptions#TX_BATCH_WRITE}
+     */
+    long getTxBatchWrite();
+
+    /**
+     * Default Value: {@value BaseIotaConfig.Defaults#MILESTONE_BATCH_WRITE}
+     *
+     * @return {@value DbConfig.Descriptions#MILESTONE_BATCH_WRITE}
+     */
+    int getMilestoneBatchWrite();
```

Comment:
I meant something a bit different in the issue for those configs to do...
But I think we shouldn't do that to not upset too many changes.. So I'll comment on the issue a bit later on

So change this to `cacheSize`  since it reflects what this config does